packages/synapse-react-client/src/components/ChallengeDataDownload/Renderers.tsx

```diff
@@ -35,9 +35,10 @@ export function DownloadRenderer<T extends EntityIdAndVersionNumber>(
   props: CellRendererProps<T>,
 ) {
   return (
-    <DirectDownload
-      associatedObjectId={props.rowData.entityId}
+    <FileEntityDirectDownload
+      entityId={props.rowData.entityId}
+      entityVersionNumber={props.rowData.versionNumber}
       stopPropagation={true}
-    ></DirectDownload>
+    />
```

Comment:
`DirectDownload` was changed to no longer accept entity IDs to reduce complexity. `FileEntityDirectDownload` was created to handle this case, so use it here.